apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.cs

```diff
@@ -147,264 +145,20 @@ public UrlSigner WithSigningVersion(SigningVersion signingVersion)
         /// in a context that should not block.
         /// </para>
         /// <para>
-        /// When the <paramref name="request"/> is specified, some of its headers will be included in the signed URL's
```

Comment:
I'm not sure about removing the existing public methods. Could add a method accepting a RequestTemplate but keep the existing ones, or at least some of them? While we definitely *can* make breaking changes, if there are easy ways of not breaking existing code without creating a significant code smell, I think that would be useful.